dev-guide/featuresets.md

```diff
@@ -403,6 +403,23 @@ The biggest difference is that our configuration is one of our APIs.
 Configuration never goes away.
 Feature Gates can and do disappear by design.
 
+## Can you disable a Feature Gate that is enabled?
+This depends on what you mean by the questions.
```

Comment:
```suggestion
This depends on what you mean by the question.

If you're asking whether you can disable a Feature Gate on a cluster after enabling it, then the answer is no.  The "enabled" and "disabled" fields in the Feature Gate are not meant to be a toggle selection.
```

Reply:
> If you're asking whether you can disable a Feature Gate on a cluster after enabling it, then the answer is no.  You can enable a feature by patching the cluster feature gate, but you cannot disable it by patching the cluster feature gate. 


@deads2k this blurb is the main reason I asked for clarification.  Is this right way to express this, or even correct?  We've not been able to toggle the state from enabled back to disabled, but it's never been documented anywhere.